cinema/views.py

```diff
@@ -358,19 +341,20 @@ def get_context_data(self, **kwargs):
 
 
 def film_detail(request, pk):
-    film_list = Film.objects.select_related("imdb_rating", "mpaa_rating")
+    """
+    Display page with details of movie selected by visitor.
+
+    Also display comments to movie and form for adding new comment.
+    Create "CommentToFilm" model record.
+    After saving data, redirect to current page.
+    """
+    film_list = Film.films.get_queryset()
```

Comment:
Could you hide `get_queryset` into `all` method. It should be `Film.films.all()`

Reply:
ok